text/3488-deprecate-cargo-categories.md

```diff
@@ -0,0 +1,374 @@
+original categories implementation pull request [here](https://github.com/rust-lang/crates.io/pull/488#issuecomment-265593195)
+and [here](https://github.com/rust-lang/crates.io/pull/488#issuecomment-265623340).
+
+**Comparison:**
+
+- **Confusion:** Having both categories and keywords can be confusing to users
+due to the [overlap](https://github.com/rust-lang/crates.io/pull/488#issuecomment-265593195)
+of functionality in both concepts. Especially when searching for crates, it is
+not clear whether it's better to look for a matching category or a matching
+keyword. With just keywords it can potentially be confusing too if multiple
+matching keywords exist, though that can be countered to some degree by
+promoting one of them to a recommended keyword.
+
+- **Creation:** One big advantage of the "recommended keywords" concept is that
+keywords can be promoted to be "recommended" retroactively
+[based on usage data](https://github.com/rust-lang/crates.io/pull/488#issuecomment-265597304).
+Categories need to be created by the crates.io team before they can be used by
+any crates, which leads to situations where such categories are not actually
+being used and [only contain a single crate](https://crates.io/categories/science::neuroscience).
+
+- **Duplicates:** With a list of recommended keywords the chance for duplicates
+can be reduced, but not eliminated completely. Categories have the advantage
+of not having that problem, but at the same time they are also restrict crate
+authors to only those categories that have been pre-selected by the crates.io
+team.
+
+- **Clarity:** Similar to reducing the number of duplicates, having a list of
+recommended keywords with associated descriptions can help improve clarity
+on which crates should be labeled with certain keywords. The same applies to
+categories as well.
+
+- **Politics:** As discussed above, the introduction of a category is being seen
+as a political decision by some users. If recommended keywords are promoted
+primarily based on usage data then the political aspect does not come into
+play quite as much, reducing the psychological pressure on the crates.io team.
+
+- **Grassroots:** When keywords are promoted to being recommended based
+primarily on usage data then there is less perception of "top-down
+decision-making" compared to the fixed list of categories "dictated" by the
+crates.io team.
+
+- **Choice:** Choosing from the list of all available keywords can quickly lead
+to choice overload. Having recommended keywords can counter this by giving
+crate authors a short list of keywords to choose from, and only if they don't
+find any fitting keywords they can drop down to the full list of keywords or
+introduce a new one.
+
+- **Bike-shedding:** As with previous points in this list, promoting keywords
+based on usage data largely avoids the bike-shedding discussions that happen
+when adjusting the list of categories.
```

Comment:
Since this is slightly more accurate 😉

```suggestion
- **Bike-shedding:** As with previous points in this list, promoting keywords
based on usage data means that any bike-shedding discussions on what keywords to
use have already happened by users, and the crates.io team only has to partake
in the logistics of adding information about existing tags.
```